src/dsp_tools/xmllib/models/res.py

```diff
@@ -1504,6 +1509,10 @@ def add_file(
 
         Args:
             filename: path to the file
+            license: License of the file (predefined or custom).
```

Comment:
"License of the file (predefined or custom)." does not add new information, it's just repetition of what is already stated in the signature ("license: PreDefinedLicense | str")